### PR TITLE
Removes GoogleSheet Integration and Updates README to Provide Tradeoffs

### PR DESCRIPTION
**Pull request by paladp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,7 +140,35 @@ represented as true and false.  Round-robin randomizes the categories and
...
+```
+scores: [4, 6, 4, 2, 2, 2]
+average: 3
+```
+
+In this case, the average score is the same as with the random grouping method,
+but all the groups have atleast one student willing to be a team leader. This
+has the potential to make the assignment more effective by maximizing team
+effectiveness.
+
```
Comment (aubreypc) [inline README.md]: mdl error:
README.md:172: MD012 Multiple consecutive blank lines

File: README.md [hunk 2]
```diff
@@ -136,7 +140,35 @@ represented as true and false.  Round-robin randomizes the categories and
...
+assignment. Consider using this method for assignments where students might have
+specialized roles. Take for example a poll that asks the students if they would
+be interested in taking on more responsibility as a team leader. Using the
+random method and the -v flag to see additional output, GatorGrouper may produce
+an output like this:
+
+```
+scores: [4, 4, 2, 0, 6, 4]
+average: 3
+```
```
Comment (aubreypc) [inline README.md]: README.md:155: MD031 Fenced code blocks should be surrounded by blank lines

**Comment by paladp** [comment]
@aubreypc New commit addresses those issue. I did not mean to have this PR onto the master, since I didn't know what we wanted to do regarding the restructuring, but I'll let you decide on what the best course of action is.

**Comment by Lancasterwu** [comment]
I removed the flake8 to see if it will pass Travis CI.  It should be able to. Will review the code in details this weekend.

**Comment by gkapfham** [comment]
Hello @Lancasterwu can you please give a status update on the review that you did over the weekend? I would like to know whether or not it is possible to merge this into the master branch. Also, what is the plan for this PR? Are you going to merge it into the master branch and then ask all of the other open PRs to merge this?

**Comment by paladp** [comment]
Waiting until issue #142 has merged because it fixes the import errors

**Comment by paladp** [comment]
Also, found an error after the merge from master, I'm trying to fix that, I'll comment here when I've fixed it

**Comment by Michionlion** [comment]
Closing this PR as #154 merges these changes.

**Pull request closed without merging**